linera-execution/src/system.rs

```diff
@@ -191,6 +195,10 @@ pub enum SystemOperation {
     },
     /// Operations that are only allowed on the admin chain.
     Admin(AdminOperation),
+    /// Processes an event about a new epoch and committee.
+    ProcessNewEpoch(Epoch),
+    /// Processes an event about a removed epoch and committee.
+    ProcessRemovedEpoch(Epoch),
```

Comment:
Why do we even need this? I.e. creating a new epoch (and changing the committee) should "remove" the previous epoch?

Reply:
Multiple epochs can be trusted at the same time. Removing them means no longer trusting them. In the future this will have to be based on an expiry time instead.